### PR TITLE
Import first bunch of features for NanoAOD v2 (same as 94XAN #22679)

### PR DESCRIPTION
**Pull request by arizzi**
-minor fixes (genlevel masses of bosons and isEE is EB)
-era switches for MiniAOD V1 vs V2 inputs

fwd port of #22679

### CODE REVIEW

File: Configuration/StandardSequences/python/Eras.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def __init__(self):
                            'phase2_hgcal', 'phase2_muon', 'phase2_timing',
                            'phase2_timing_layer','phase2_hcal',
                            'trackingLowPU', 'trackingPhase1', 'ctpps_2016', 'trackingPhase2PU140',
-                           'tracker_apv_vfp30_2016', 'run2_miniAOD_80XLegacy','run2_miniAOD_94XFall17', 'run2_nanoAOD_92X',
+                           'tracker_apv_vfp30_2016', 'run2_miniAOD_80XLegacy','run2_miniAOD_94XFall17', 'run2_nanoAOD_92X',  'run2_nanoAOD_94XMiniAODv1', 'run2_nanoAOD_94XMiniAODv2',
```
Comment (slava77) [inline Configuration/StandardSequences/python/Eras.py]: please add a line break

**Comment by arizzi** [comment]
+1

**Comment by gpetruc** [comment]
The error is
````
----- Begin Fatal Exception 21-Mar-2018 13:55:01 CET-----------------------
An exception of category 'fastsim::ParticleManager' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 63 stream: 2
   [1] Running path 'simulation_step'
   [2] Calling method for module FastSimProducer/'fastSimProducer'
Exception Message:
unknown pdg id: 100441
----- End Fatal Exception -------------------------------------------------
````
which doesn't look related to this PR in any way.

**Comment by fabiocos** [comment]
please test

**Comment by fabiocos** [comment]
given the discussion in #22679 I would say that the present implementation is reasonable for the time, being, also with the target of keeping in sync 9_4 and the master. On a longer term some general cleaning should be done along David's suggestion

The failure in the fastsim addOn test looks unrelated to this PR, it did not appear in the IB, although it has been reproduced.

**Comment by fabiocos** [comment]
+1

**Comment by fabiocos** [comment]
merge

**Pull request merged**